aws/table_aws_s3_bucket.go

```diff
@@ -685,6 +696,32 @@ func getBucketTagging(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrat
 	return bucketTags, nil
 }
 
+func getBucketWebsite(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	if h.HydrateResults["getBucketLocation"] == nil {
+		return nil, nil
```

Comment:
Please take a look at the additional comments that have been added in other hydrate functions and add that in if applicable.

```
	// Bucket location will be nil if getBucketLocation returned an error but
	// was ignored through ignore_error_codes config arg
```